massa-proto/proto/massa/api/v1/versioning.proto

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2023 MASSA LABS <info@massa.net>
+
+syntax = "proto3";
+
+package massa.api.v1;
+
+option csharp_namespace = "Com.Massa.Api.V1";
+option go_package = "github.com/massalabs/massa/api/v1;v1";
+option java_multiple_files = true;
+option java_package = "com.massa.api.v1";
+option objc_class_prefix = "GRPC";
+option php_namespace = "Com\\Massa\\Api\\V1";
+option ruby_package = "Com::Massa::Api::V1";
+option swift_prefix = "GRPC";
+
+// Entry for GetMipStatusResponse
+message MipStatusEntry {
+  // Mip info
+  MipInfo mip_info = 1;
+  // state id
+  ComponentStateId state = 2;
```

Comment:
ssame here `state_id`

Reply:
I've just rename the field name to state_id. Moving it to the top does not make sense because we want here to have a mapping of Mip info -> state_id